packages/embed/src/utils/url-parameters-transfer.ts

```diff
@@ -0,0 +1,32 @@
+type Params = { [key: string]: string }
+
+type QueryStrings = {
+  [key: string]: any
+}
```

Comment:
Please put type declarations below the meaningful code

Reply:
If you use built-in `URL` type you might not need custom types at all.